site/content/docs/5.2/components/placeholders.md

```diff
@@ -9,6 +9,11 @@ toc: true
 added: "5.1"
 ---
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This component variant should not be used because it does not exist in the Orange Design System specifications.
```

Comment:
It was worked directly with the designers. They said at the time that it would be a great addition for ODS. I wouldn't display this callout.

Reply:
Yes, they should have a card in their board for that.
If the design in Boosted is OK because done with designers => OK to delete the callout now. 
Otherwise, I think it is better to keep the callout and delete it afterwards. So far, we applied this same rule everywhere...

Reply:
Resolved ?